queenbee/schema/metadata.py

```diff
@@ -48,9 +49,9 @@ class MetaData(BaseModel):
         description='A short description of the workflow.'
     )
 
-    contact: Contact = Field(
+    author: Union[Author, List[Author]] = Field(
```

Comment:
Oh hadn't caught this when reviewing your latest changes. Can we just have it as one type: `List[Author]`? I'm keen to reduce unions to where they are absolutely necessary to reduce overall schema complexity.

Reply:
Sounds good. I will push a fix!